src/views/home.ejs

```diff
@@ -0,0 +1,16 @@
+<%- include('partials/header') %>
+
+
+<div class="jumbotron centered">
+  <div class="container">
+    <i class="fas fa-key fa-6x"></i>
+    <h1 class="display-3">HuskyAPI</h1>
+    <p class="lead">An API for the University of Washington System that contains Tri-Campus Course Data!</p>
```

Comment:
This might be covered in the Confluence board but we should add a description of what's supported in the moment (ex. Only Bothell courses at the moment) once we do push this and have people other than us testing it

Reply:
Good catch! Will update.